docs/kernel_finder.rst

```diff
@@ -0,0 +1,183 @@
+.. _kernel_finder:
+
+================
+Kernel Discovery
+================
+
+The primary purpose of the Jupyter Kernel management package is to provide
+a means of discovering kernels that are available for use.  This is accomplished
+using the :class:`KernelFinder <jupyter_kernel_mgmt.discovery.KernelFinder>` class.
+
+:class:`KernelFinder <jupyter_kernel_mgmt.discovery.KernelFinder>` instances are created in one of two ways.
+
+1. The most common way is to call KernelFinder's class method 
+:meth:`.KernelFinder.from_entrypoints()`. This loads all of the registered
+kernel providers.
```

Comment:
Have you built this? I thought rst needed the following lines to be indented to continue the numbered list entry.

Reply:
Yes - I built this (dozens of times) and it appears to render fine.